frontend/src/App.js

```diff
@@ -1,23 +1,24 @@
-import logo from './logo.svg';
+import React from "react";
+import { Route, Routes } from 'react-router-dom';
 import './App.css';
+import Navbar from "./components/Navbar/Navbar";
+import Home from './routes/Home';
+import Threads from './routes/Threads';
+import ThreadView from './routes/ThreadView';
+import YourProfile from './routes/YourProfile';
+import Settings from './routes/Settings';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Navbar/>
```

Comment:
Why is Navbar outside of Route; Also I feel like BrowserRouter should be placed here on top of routes.

Reply:
Ok, I can fix this